zebra/if_netlink.c

```diff
@@ -903,8 +903,8 @@ int kernel_interface_set_master(struct interface *master,
 
 	req.ifa.ifi_index = slave->ifindex;
 
-	addattr_l(&req.n, sizeof(req), IFLA_MASTER, &master->ifindex, 4);
-	addattr_l(&req.n, sizeof(req), IFLA_LINK, &slave->ifindex, 4);
+	nl_attr_put(&req.n, sizeof(req), IFLA_MASTER, &master->ifindex, 4);
+	nl_attr_put(&req.n, sizeof(req), IFLA_LINK, &slave->ifindex, 4);
```

Comment:
there are several of these - any benefit to cleaning them up?

Reply:
I did some changes to most places where `addattr` was called, so I used this opportunity to rename those functions to more be self-descriptive. If you find this modification pointless, it won't be a problem to revert this.

Reply:
I meant the places where "put32" could have been used, but raw "put" with size "4" was being used?

Reply:
Yeah, you are totally right. I will change this.